src/odemis/gui/cont/streams.py

```diff
@@ -346,9 +346,7 @@ def _add_hw_setting_controls(self):
 
         # Process the hardware VAs first (emitter and detector hardware VAs are combined into one
         # attribute called 'hw_vas'
-        # sort the list of hardware VAs first in case it contains items that are not in the
-        # emitter and detector conf, they will be appended at the end
-        vas_names = util.sorted_according_to(sorted(list(self.stream.hw_vas.keys())),
+        vas_names = util.sorted_according_to(list(self.stream.hw_vas.keys()),
```

Comment:
Why remove that part? It shouldn't matter for your goal, right? In general, the idea was to make sure that the VAs always are in the same order, even the ones which are not in the HW_CONFIG.